cuda_bindings/docs/source/overview.md

```diff
@@ -312,6 +312,22 @@ maximize performance ({numref}`Figure 1`).
 Screenshot of Nsight Compute CLI output of CUDA Python example.
 ```
 
+## Getting the address of underlying C objects from the low-level bindings
```

Comment:
This section is not used anywhere in this page. Can we move it to a new page, say "Q&A", "Tips & tricks", "Common pitfalls", etc, so that we can collect this and other pending doc updates in one easy-to-find place for binding users?